check_pgactivity

```diff
@@ -7264,12 +7266,16 @@ SELECT
                 $sequences{$seq}->{ABSVALSEQ}=-$sequences{$seq}->{LASTVALSEQ};
             }
             # The real maximum value is the smallest of both
-            my $real_max_value=$max_val_seq<=$max_value?$max_val_seq:$max_value;
+	    my $real_max_value;
+	    if ( defined $max_value ) {
+                $real_max_value=$max_val_seq<=$max_value?$max_val_seq:$max_value;
```

Comment:
if you're going to touch that line, maybe we could add space around operator? it's a bit unreadable as-is

Reply:
done in 73caa656410f68e02c2dcbc4499b3bd3add77fa2